Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py

```diff
@@ -530,6 +575,40 @@ def get_external_ip_address_range_command(
     return command_results
 
 
+def get_attack_surface_rule_command(
+    args: Dict[str, Any], client: Client
+) -> CommandResults:
+    """
+    asm-get-attack-surface-rule command: Returns attack surface rule details.
+
+    Args:
+        client (Client): CortexAttackSurfaceManagment client to use.
+        args (dict): all command arguments, usually passed from ``demisto.args()``
+
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains Remediation guidance information.
+    """
+    attack_surface_rule_id = str(args.get("attack_surface_rule_id"))
+    enabled_status = str(args.get("enabled_status"))
+    priority = str(args.get("priority"))
+    category = str(args.get("category"))
+
+    response = client.get_attack_surface_rule_request(attack_surface_rule_id, enabled_status, priority, category)
+    if response is not None:
```

Comment:
This part can be simplified: https://github.com/demisto/content/blob/2708baeecc740d781bdf39647073d62172aec28d/Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.py#L346

Reply:
Updated: 0ad1560d46701be1f51498172d3236454085f87f

Reply:
looks good